.bash_history

```diff
@@ -0,0 +1,3 @@
+BOARD=bluepill-128kib make -j8 -C examples/suit_update/
+quit
+exit
```

Comment:
Where did that come from?

Reply:
Oops, added by mistake after using riotdocker. I removed it.